ckan/tests/logic/action/test_patch.py

```diff
@@ -234,3 +235,26 @@ def test_resource_patch_for_update(self):
         with mock.patch.dict('ckan.logic._actions', {'package_show': mock_package_show}):
             helpers.call_action('resource_patch', id=resource['id'], description='hey')
             assert mock_package_show.call_args_list[0][0][0].get('for_update') is True
+
+    def test_user_patch_sysadmin(self):
```

Comment:
Can you add a test for sysadmins being able to change another user sysadmin status? Just for completeness

Reply:
extra assertions have been added